classes/controller/menu/class-postmeta-check-controller.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Post meta duplicate records check class
+ *
+ * @package P4BKS\Controllers\Menu
+ */
+
+namespace P4GBKS\Controllers\Menu;
+
+use P4GBKS\Command\Duplicated_Postmeta;
+
+/**
+ * Class Postmeta_Check_Controller
+ */
+class Postmeta_Check_Controller extends Controller {
+	/**
+	 * Create menu/submenu entry.
+	 */
+	public function create_admin_menu() {
+		$current_user = wp_get_current_user();
+		if ( in_array( 'administrator', $current_user->roles, true ) && current_user_can( 'manage_options' ) ) {
```

Comment:
Won't an administrator by definition always have all capabilities?

Reply:
Yes he has. The second check is't needed here.